### DISCUSSION TITLE
Is DependencyAttribute still a planned feature?

### DISCUSSION THREAD

**Discussion by senketsu03** [nunit/nunit#4397] [Q&A] [answered]
There was not any activity in #51 for years, so I decided to ask here

**Comment by senketsu03** [comment]
`[Order]` attribute isn't very helpful in my case. I'm writing integration tests for my db. Suppose I have a table `users`, `courses` and `users_courses`. As the name suggests, the `users_courses` table depends on PKs from `users` and `courses` tables, therefore the test to push data into `users_courses` table before both of them contain data fails. `[Order]` attribute applies only for methods in one class, so there's no point in using it in my case at all

**Reply by OsirisTerje** [reply, marked as answer]
Short answer, No, it is not a planned feature.  We have the Order attribute, and no one has taken it upon themselves to do anything more. 

> [Order] attribute applies only for methods in one class, so there's no point in using it in my case at all

A test in NUnit is always a method, how are you testing without it?

**Reply by mikkelbu** [reply]
@senketsu03 Perhaps you can use https://github.com/Sebazzz/NUnitTestOrdering . I've no idea of the status of it, but as far as I know this one is quite flexible (and the only 3rd party library for test ordering).

**Reply by senketsu03** [reply]
Thank you for your suggestion, I'll take a look at that!
Meanwhile I gave some thoughts in reimplementing my tests architecture, as @OsirisTerje suggested (thanks for advice!), so I think I'll rewrite my code the way I won't need test dependencies.
